src/js/components/container/app.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import ImageList from "../presentational/ImageList.jsx";
+import BulletList from "../presentational/BulletList.jsx";
+import axios from "axios";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentProduct: 1,
+      data: []
+    };
+  }
+
+  componentDidMount() {
```

Comment:
What if you moved lines 16-21 into their own method that gets invoked here? Modularizes your code a bit and gives you a function to use if you ever need to fetch the data again.

Reply:
Yeah, good idea. I was thinking I should do that after reviewing your code. I will refactor and incorporate db helper functions.